courses/fundamentals_of_ada/030_basic_types.rst

```diff
@@ -843,6 +847,28 @@ Floating Point Type Attributes
    - Advanced machine representation of the floating-point type
    - Mantissa, strict mode
 
+---------------------------
+Numeric Types Conversion
+---------------------------
+
+* Ada's integer and real are :dfn:`numeric`
+
+    - Holding a numeric value
+
+* Special rule: they can always be converted inbetween them
+
+    - Explicitely
+    - :ada:`Real` |rightarrow| :ada:`Integer` causes **rounding**
```

Comment:
Do we want want to mention rounding occurs away from 0? (e.g. 1.5 => 2, -1.5 => -2)
Or is that to much info?

Reply:
Yes this is important, I added it to the `Float'Rounding` description (previous slide)